### PR TITLE
Admin fr strings

### LINKED ISSUE #1704: Admin interface directory in French
# Issue/opportunity

Admin interface directory in French
Strings missings from translation files

# Screenshots

Screenshot 2023-03-07 at 1 54 30 PM

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

Fixes #1704

Cleans up remaining missing strings and updates admin to have a left nav

- Added left nav 
- Redirects from Welcome page to users by default

Screenshot 2023-05-31 at 11 59 24 AM

Also updated the admin login to bring it inline with the rest of the login pages
Screenshot 2023-05-31 at 12 00 10 PM

# Test instructions | Instructions pour tester la modification
- Login and test CDS admin

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [ ] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [ ] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [ ] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

**Review by craigzour — COMMENTED** [review]
Reviewed and tested. Will not approve right now as I want to make sure the new approach with the redirect depending on permissions is approved by the rest of the team ;)

**Comment by timarney** [comment]
Pushed the update to re-add view templates for @srtalbot

**Review by craigzour — CHANGES_REQUESTED** [review]

File: components/admin/LeftNav/LeftNavigation.tsx [hunk 1]
```diff
@@ -26,6 +32,19 @@ export const LeftNavigation = () => {
         </LeftNavLink>
       )}
 
+      {ability?.can("update", "Privilege") && (
```
[the remark this answers is not in the document]
Comment (timarney) [reply components/admin/LeftNav/LeftNavigation.tsx]: Pushed an update --- let me know if that'S correct.  Feel free to push a commit.

**Review by craigzour — APPROVED** [review]
LGTM :) Nice improvements!

**Pull request merged**